src/plugin/grpc-plugin.ts

```diff
@@ -79,6 +79,7 @@ export class GrpcPlugin {
       url: req.url,
       width: req.width,
       height: req.height,
+      scale: (req.scale > 0 && req.scale != Infinity) ? req.scale : 1,
```

Comment:
Might it be better to drop this validation, since we also validate later? If we were to change the default later f.ex., it might be bad to hardcode it twice. Bugs could also potentially arise from having two different validation logics.

Reply:
You are right about this.